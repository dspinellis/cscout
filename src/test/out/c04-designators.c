Table: Ids
EID	NAME	READONLY	UNDEFMACRO	MACRO	MACROARG	ORDINARY	SUETAG	SUMEMBER	LABEL	TYPEDEF	ENUM	YACC	FUN	CSCOPE	LSCOPE	UNUSED
79	foo	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1
91	MAXDIGIT	0	0	1	0	0	0	0	0	0	0	0	0	0	0	1
199	a	0	0	0	0	0	0	1	0	0	0	0	0	0	0	1
229	b	0	0	0	0	0	0	1	0	0	0	0	0	0	0	1
331	x	0	0	0	0	1	0	0	0	0	0	0	0	0	1	1
349	bar	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1
441	main	1	0	0	0	1	0	0	0	0	0	0	1	0	1	0
449	k	0	0	0	0	0	0	1	0	0	0	0	0	0	0	1
499	l	0	0	0	0	0	0	1	0	0	0	0	0	0	0	1
559	a	0	0	0	0	1	0	0	0	0	0	0	0	0	1	1
571	qqq	0	0	0	0	1	0	0	0	0	0	0	0	0	1	1
829	foo2	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0
891	foo	0	0	0	0	1	0	0	0	0	0	0	1	0	1	1
959	X	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0
989	b	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0
1109	bar2	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1
1219	k	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0
1269	l	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0
1359	c	0	0	0	0	1	0	0	0	0	0	0	0	0	1	1
1889	d	0	0	0	0	1	0	0	0	0	0	0	0	0	1	1
2679	b	0	0	0	0	1	0	0	0	0	0	0	0	0	1	1
2975	__DATE__	1	0	1	0	0	0	0	0	0	0	0	0	0	0	1
3295	__TIME__	1	0	1	0	0	0	0	0	0	0	0	0	0	0	1
3575	__FILE__	1	0	1	0	0	0	0	0	0	0	0	0	0	0	1
3865	__LINE__	1	0	1	0	0	0	0	0	0	0	0	0	0	0	1
4055	__STDC__	1	0	1	0	0	0	0	0	0	0	0	0	0	0	1
5355	_cscout_dummy1	1	0	0	0	1	0	0	0	0	0	0	1	1	0	0
Table: Tokens
FID	FOFFSET	EID
2	297	2975
2	329	3295
2	357	3575
2	386	3865
2	405	4055
2	471	441
2	535	5355
2	558	5355
4	7	79
4	19	199
4	22	229
4	34	349
4	44	449
4	49	499
4	55	559
4	82	829
4	95	959
4	98	989
4	110	1109
4	121	1219
4	126	1269
4	135	1359
4	142	959
4	150	989
4	158	1269
4	166	1219
4	177	1219
4	188	1889
4	195	959
4	203	989
4	211	1269
4	225	1219
4	242	1219
4	262	829
4	267	2679
4	273	989
4	281	1269
4	286	1219
4	294	959
5	8	91
5	32	331
5	43	441
5	56	571
5	88	891
Table: Rest
FID	FOFFSET	CODE
2	287	\u0000a\u0000a#define 
2	305	  
2	320	\u0000a#define 
2	337	 
2	348	\u0000a#define 
2	365	 
2	377	\u0000a#define 
2	394	 1\u0000a#define 
2	413	 1\u0000a\u0000a
2	466	\u0000aint 
2	475	();\u0000a
2	522	\u0000astatic void 
2	549	(void) { 
2	572	(); }\u0000a
3	152	\u0000a\u0000a\u0000a#pragma includepath 
3	195	\u0000a
3	239	\u0000astatic void _cscout_dummy2(void) { _cscout_dummy2(); }\u0000a
4	0	struct 
4	10	 {\u0000d\u0000a	int 
4	20	, 
4	23	;\u0000d\u0000a	struct 
4	37	 { int 
4	45	; } 
4	50	;\u0000d\u0000a} 
4	56	 = {1, 2, {3}};\u0000d\u0000a\u0000d\u0000astruct 
4	86	 {\u0000d\u0000a	int 
4	96	, 
4	99	;\u0000d\u0000a	struct 
4	114	 { int 
4	122	; } 
4	127	[3];\u0000d\u0000a} 
4	136	 = { .
4	143	 = 1, .
4	151	 = 2, .
4	159	 = {{ .
4	167	 = 3}, { .
4	178	 = 4}}},\u0000d\u0000a
4	189	 = { .
4	196	 = 1, .
4	204	 = 2, .
4	212	 = {[0] = { .
4	226	 = 3}, [1] = { .
4	243	 = 4}}};\u0000d\u0000a\u0000d\u0000astruct 
4	266	 
4	268	 = {.
4	274	 = 1, .
4	282	[2].
4	287	 = 3, .
4	295	 = 8};\u0000d\u0000a
5	0	#define 
5	16	 11\u0000d\u0000aextern int 
5	33	;\u0000d\u0000aextern 
5	47	();\u0000d\u0000aint 
5	59	;\u0000d\u0000a\u0000d\u0000a
5	86	\u0000d\u0000a
5	91	() {\u0000d\u0000a	
5	116	}\u0000d\u0000a\u0000d\u0000a
Table: Projects
PID	NAME
16	unspecified
17	Prj1
18	Prj2
Table: IdProj
EID	PID
79	17
199	17
229	17
349	17
441	17
449	17
499	17
559	17
829	17
959	17
989	17
1109	17
1219	17
1269	17
1359	17
1889	17
2679	17
2975	17
3295	17
3575	17
3865	17
4055	17
5355	17
79	18
91	18
199	18
229	18
331	18
349	18
441	18
449	18
499	18
559	18
571	18
829	18
891	18
959	18
989	18
1109	18
1219	18
1269	18
1359	18
1889	18
2679	18
2975	18
3295	18
3575	18
3865	18
4055	18
5355	18
Table: Files
FID	NAME	RO
2	host-defs.h	1
3	host-incs.h	1
4	c04-designators.c	0
5	prj2.c	0
Table: Filemetrics
FID	PRECPP	NCHAR	NCCOMMENT	NSPACE	NLCOMMENT	NBCOMMENT	NLINE	MAXLINELEN	MAXSTMTLEN	MAXSTMTNEST	MAXBRACENEST	MAXBRACKNEST	NULINE	NPPDIRECTIVE	NPPCOND	NPPFMACRO	NPPOMACRO	NTOKEN	NSTMT	NOP	NUOP	NNCONST	NCLIT	NSTRING	NIF	NELSE	NSWITCH	NCASE	NDEFAULT	NBREAK	NFOR	NWHILE	NDO	NCONTINUE	NGOTO	NRETURN	NPID	NFID	NMID	NID	NUPID	NUFID	NUMID	NUID	NLABEL	NCOPIES	NINCFILE	NPFUNCTION	NFFUNCTION	NPVAR	NFVAR	NAGGREGATE	NAMEMBER	NENUM	NEMEMBER
2	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	16	0	1	1	NULL	NULL	NULL	NULL	NULL	18	2	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	NULL	0	0	0	NULL	0	0	NULL	NULL	0	1	0	0	0	0	0	0
2	1	578	367	29	0	3	22	61	47	NULL	1	1	0	5	0	0	5	37	2	0	0	2	0	3	0	0	0	0	0	0	0	0	0	0	0	0	1	2	5	8	1	1	5	7	0	1	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	NULL
3	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	15	0	1	1	NULL	NULL	NULL	NULL	NULL	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	NULL	0	0	0	NULL	0	0	NULL	NULL	0	0	0	0	0	0	0	0
3	1	295	187	13	0	2	13	54	28	NULL	1	1	0	1	0	0	0	16	1	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	NULL
4	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	120	0	3	0	NULL	NULL	NULL	NULL	NULL	152	9	33	0	18	0	0	0	0	0	0	0	0	0	0	0	0	0	0	8	0	NULL	8	4	0	NULL	4	0	NULL	NULL	0	0	4	0	4	8	0	0
4	1	303	0	106	0	0	12	64	120	NULL	3	0	0	0	0	0	0	151	9	33	0	18	0	0	0	0	0	0	0	0	0	0	0	0	0	0	4	0	0	4	4	0	0	4	0	1	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	NULL
5	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	8	0	1	1	NULL	NULL	NULL	NULL	NULL	18	3	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	4	0	NULL	4	4	0	NULL	4	0	NULL	NULL	1	0	2	0	0	0	0	0
5	1	121	34	26	2	0	11	21	14	NULL	1	1	0	1	0	0	1	21	3	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	4	0	1	5	4	0	1	5	0	1	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	NULL
Table: FileProj
FID	PID
2	17
3	17
4	17
1	18
2	18
3	18
4	18
5	18
Table: Definers
PID	CUID	BASEFILEID	DEFINERID
18	5	5	2
Table: Includers
PID	CUID	BASEFILEID	INCLUDERID
17	2	2	1
17	4	3	1
17	4	4	1
18	2	2	1
18	2	2	1
18	4	3	1
18	4	4	1
18	5	3	1
18	5	5	1
Table: Providers
PID	CUID	PROVIDERID
17	2	2
17	4	4
18	2	2
18	2	2
18	4	4
18	5	5
Table: IncTriggers
PID	CUID	BASEFILEID	DEFINERID	FOFFSET	LEN
18	5	5	2	471	4
Table: Functions
ID	NAME	ISMACRO	DEFINED	DECLARED	FILESCOPED	FID	FOFFSET	FANIN
891	foo	0	1	1	0	5	88	0
4715	main	0	0	1	0	2	471	0
5355	_cscout_dummy1	0	1	1	1	2	535	1
Table: FunctionDefs
FUNCTIONID	FIDBEGIN	FOFFSETBEGIN	FIDEND	FOFFSETEND
-3126566792108663513	2	557	2	577
6878446942784652761	5	95	5	117
Table: FunctionMetrics
FUNCTIONID	PRECPP	NCHAR	NCCOMMENT	NSPACE	NLCOMMENT	NBCOMMENT	NLINE	MAXLINELEN	MAXSTMTLEN	MAXSTMTNEST	MAXBRACENEST	MAXBRACKNEST	NULINE	NPPDIRECTIVE	NPPCOND	NPPFMACRO	NPPOMACRO	NTOKEN	NSTMT	NOP	NUOP	NNCONST	NCLIT	NSTRING	NIF	NELSE	NSWITCH	NCASE	NDEFAULT	NBREAK	NFOR	NWHILE	NDO	NCONTINUE	NGOTO	NRETURN	NPID	NFID	NMID	NID	NUPID	NUFID	NUMID	NUID	NLABEL	NGNSOC	NMPARAM	NFPARAM	FANIN	FANOUT	CCYCL1	CCYCL2	CCYCL3	CSTRUC	CHAL	IFLOW
891	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	0	0	0	0	NULL	NULL	NULL	NULL	NULL	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	NULL	0	0	0	NULL	0	0	20	NULL	0	0	0	1	1	1	0.0	0.0	0.0
891	1	23	15	4	1	0	2	18	0	NULL	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	NULL	0	NULL	0	0	1	1	1	0.0	0.0	0.0
5355	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	5	0	0	1	NULL	NULL	NULL	NULL	NULL	5	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	NULL	1	0	1	NULL	1	0	12	NULL	0	1	1	1	1	1	1.0	0.0	1.0
5355	1	21	0	3	0	0	1	20	5	NULL	0	1	0	0	0	0	0	5	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	1	0	1	0	1	0	NULL	0	NULL	1	1	1	1	1	1.0	0.0	1.0
Table: FunctionId
FUNCTIONID	ORDINAL	EID
891	0	891
4715	0	441
5355	0	5355
Table: Fcalls
SOURCEID	DESTID
5355	5355
Done
