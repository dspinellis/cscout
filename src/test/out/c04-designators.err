Processing workspace TestWS
Processing directory test/c
Entering directory test/c
Processing project Prj1
Processing file c04-designators.c
Done processing file c04-designators.c
Done processing project Prj1
Processing project Prj2
Processing file c04-designators.c
Done processing file c04-designators.c
Processing file prj2.c
Done processing file prj2.c
Done processing project Prj2
Exiting directory test/c
Done processing directory test/c
Done processing workspace TestWS
Processing identifiers
3%6%10%13%17%20%24%27%31%34%37%41%44%48%51%55%58%62%65%68%72%75%79%82%86%89%93%96%100%
