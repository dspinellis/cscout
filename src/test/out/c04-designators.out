Table: Ids
EID	NAME	READONLY	UNDEFMACRO	MACRO	FUNMACRO	MACROARG	CPPCONST	CPPSTRVAL	DEFCCONSTVAL	NOTDEFCCONSTVAL	EXPCCONSTVAL	NOTEXPCCONSTVAL	ORDINARY	SUETAG	SUMEMBER	LABEL	TYPEDEF	ENUM	YACC	FUN	CSCOPE	LSCOPE	UNUSED
79	foo	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1
91	MAXDIGIT	0	0	1	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	1
189	a	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	1
219	b	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	1
321	x	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1	1
329	bar	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1
421	main	1	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	1	0	1	0
429	k	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	1
479	l	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	1
529	a	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1	1
541	qqq	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1	1
779	foo2	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0
821	foo	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	1	0	1	1
899	X	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0
929	b	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0
1039	bar2	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1
1149	k	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0
1199	l	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0
1279	c	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1	1
1799	d	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1	1
2569	b	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1	1
2975	__DATE__	1	0	1	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	1
3295	__TIME__	1	0	1	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	1
3575	__FILE__	1	0	1	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	1
3865	__LINE__	1	0	1	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	1
4055	__STDC__	1	0	1	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	1
5355	_cscout_dummy1	1	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	1	1	0	0
Table: Tokens
FID	FOFFSET	EID
2	297	2975
2	329	3295
2	357	3575
2	386	3865
2	405	4055
2	471	421
2	535	5355
2	558	5355
4	7	79
4	18	189
4	21	219
4	32	329
4	42	429
4	47	479
4	52	529
4	77	779
4	89	899
4	92	929
4	103	1039
4	114	1149
4	119	1199
4	127	1279
4	134	899
4	142	929
4	150	1199
4	158	1149
4	169	1149
4	179	1799
4	186	899
4	194	929
4	202	1199
4	216	1149
4	233	1149
4	251	779
4	256	2569
4	262	929
4	270	1199
4	275	1149
4	283	899
5	8	91
5	31	321
5	41	421
5	53	541
5	81	821
Table: Rest
FID	FOFFSET	CODE
2	287	\u0000a\u0000a#define 
2	305	  
2	320	\u0000a#define 
2	337	 
2	348	\u0000a#define 
2	365	 
2	377	\u0000a#define 
2	394	 1\u0000a#define 
2	413	 1\u0000a\u0000a
2	466	\u0000aint 
2	475	();\u0000a
2	522	\u0000astatic void 
2	549	(void) { 
2	572	(); }\u0000a
3	152	\u0000a\u0000a\u0000a#pragma includepath 
3	195	\u0000a
3	239	\u0000astatic void _cscout_dummy2(void) { _cscout_dummy2(); }\u0000a
4	0	struct 
4	10	 {\u0000a	int 
4	19	, 
4	22	;\u0000a	struct 
4	35	 { int 
4	43	; } 
4	48	;\u0000a} 
4	53	 = {1, 2, {3}};\u0000a\u0000astruct 
4	81	 {\u0000a	int 
4	90	, 
4	93	;\u0000a	struct 
4	107	 { int 
4	115	; } 
4	120	[3];\u0000a} 
4	128	 = { .
4	135	 = 1, .
4	143	 = 2, .
4	151	 = {{ .
4	159	 = 3}, { .
4	170	 = 4}}},\u0000a
4	180	 = { .
4	187	 = 1, .
4	195	 = 2, .
4	203	 = {[0] = { .
4	217	 = 3}, [1] = { .
4	234	 = 4}}};\u0000a\u0000astruct 
4	255	 
4	257	 = {.
4	263	 = 1, .
4	271	[2].
4	276	 = 3, .
4	284	 = 8};\u0000a
5	0	#define 
5	16	 11\u0000aextern int 
5	32	;\u0000aextern 
5	45	();\u0000aint 
5	56	;\u0000a\u0000a
5	80	\u0000a
5	84	() {\u0000a	
5	107	}\u0000a\u0000a
Table: Projects
PID	NAME
23	unspecified
24	Prj1
25	Prj2
Table: IdProj
EID	PID
79	24
189	24
219	24
329	24
421	24
429	24
479	24
529	24
779	24
899	24
929	24
1039	24
1149	24
1199	24
1279	24
1799	24
2569	24
2975	24
3295	24
3575	24
3865	24
4055	24
5355	24
79	25
91	25
189	25
219	25
321	25
329	25
421	25
429	25
479	25
529	25
541	25
779	25
821	25
899	25
929	25
1039	25
1149	25
1199	25
1279	25
1799	25
2569	25
2975	25
3295	25
3575	25
3865	25
4055	25
5355	25
Table: Files
FID	NAME	RO
2	host-defs.h	1
3	host-incs.h	1
4	c04-designators.c	0
5	prj2.c	0
Table: Filemetrics
FID	PRECPP	NCHAR	NCCOMMENT	NSPACE	NLCOMMENT	NBCOMMENT	NLINE	MAXLINELEN	MAXSTMTLEN	MAXSTMTNEST	MAXBRACENEST	MAXBRACKNEST	BRACENEST	BRACKNEST	NULINE	NPPDIRECTIVE	NPPCOND	NPPFMACRO	NPPOMACRO	NTOKEN	NSTMT	NOP	NUOP	NNCONST	NCLIT	NSTRING	NPPCONCATOP	NPPSTRINGOP	NIF	NELSE	NSWITCH	NCASE	NDEFAULT	NBREAK	NFOR	NWHILE	NDO	NCONTINUE	NGOTO	NRETURN	NASM	NTYPEOF	NPID	NFID	NMID	NID	NUPID	NUFID	NUMID	NUID	NLABEL	NMACROEXPANDTOKEN	NCOPIES	NINCFILE	NPFUNCTION	NFFUNCTION	NPVAR	NFVAR	NAGGREGATE	NAMEMBER	NENUM	NEMEMBER
2	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	16	0	1	1	NULL	NULL	NULL	NULL	NULL	NULL	NULL	18	2	0	0	0	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	NULL	0	0	0	NULL	0	0	NULL	NULL	NULL	0	1	0	0	0	0	0	0
2	1	548	367	29	0	3	22	61	47	NULL	1	1	0	0	0	5	0	0	5	37	2	0	0	2	0	3	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	2	5	8	1	1	5	7	0	0	1	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	NULL
3	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	15	0	1	1	NULL	NULL	NULL	NULL	NULL	NULL	NULL	0	1	0	0	0	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	NULL	0	0	0	NULL	0	0	NULL	NULL	NULL	0	0	0	0	0	0	0	0
3	1	243	187	13	0	2	13	48	28	NULL	1	1	0	0	0	1	0	0	0	16	1	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	NULL
4	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	118	0	3	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	152	9	33	0	18	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	8	0	NULL	8	4	0	NULL	4	0	NULL	NULL	NULL	0	0	4	0	4	8	0	0
4	1	289	0	94	0	0	12	69	118	NULL	3	0	0	0	0	0	0	0	0	151	9	33	0	18	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	4	0	0	4	4	0	0	4	0	0	1	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	NULL
5	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	7	0	1	1	NULL	NULL	NULL	NULL	NULL	NULL	NULL	18	3	0	0	0	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	4	0	NULL	4	4	0	NULL	4	0	NULL	NULL	NULL	1	0	2	0	0	0	0	0
5	1	95	32	17	2	0	11	20	13	NULL	1	1	0	0	0	1	0	0	1	21	3	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	4	0	1	5	4	0	1	5	0	0	1	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	NULL
Table: FileProj
FID	PID
2	24
3	24
4	24
1	25
2	25
3	25
4	25
5	25
Table: LineProj
FID	PID	LNUM
2	24	1
2	24	2
2	24	3
2	24	4
2	24	5
2	24	6
2	24	7
2	24	8
2	24	9
2	24	10
2	24	11
2	24	12
2	24	13
2	24	14
2	24	15
2	24	16
2	24	17
2	24	18
2	24	19
2	24	20
2	24	21
2	24	22
3	24	1
3	24	2
3	24	3
3	24	4
3	24	5
3	24	6
3	24	7
3	24	8
3	24	9
3	24	10
3	24	11
3	24	12
3	24	13
4	24	1
4	24	2
4	24	3
4	24	4
4	24	5
4	24	6
4	24	7
4	24	8
4	24	9
4	24	10
4	24	11
4	24	12
2	25	1
2	25	2
2	25	3
2	25	4
2	25	5
2	25	6
2	25	7
2	25	8
2	25	9
2	25	10
2	25	11
2	25	12
2	25	13
2	25	14
2	25	15
2	25	16
2	25	17
2	25	18
2	25	19
2	25	20
2	25	21
2	25	22
3	25	1
3	25	2
3	25	3
3	25	4
3	25	5
3	25	6
3	25	7
3	25	8
3	25	9
3	25	10
3	25	11
3	25	12
3	25	13
4	25	1
4	25	2
4	25	3
4	25	4
4	25	5
4	25	6
4	25	7
4	25	8
4	25	9
4	25	10
4	25	11
4	25	12
5	25	1
5	25	2
5	25	3
5	25	4
5	25	5
5	25	6
5	25	7
5	25	8
5	25	9
5	25	10
5	25	11
Table: LinePos
FID	FOFFSET	LNUM
2	0	1
2	4	2
2	51	3
2	54	4
2	100	5
2	119	6
2	122	7
2	184	8
2	231	9
2	234	10
2	284	11
2	288	12
2	289	13
2	321	14
2	349	15
2	378	16
2	397	17
2	416	18
2	417	19
2	467	20
2	479	21
2	523	22
3	0	1
3	4	2
3	51	3
3	54	4
3	78	5
3	97	6
3	100	7
3	149	8
3	153	9
3	154	10
3	155	11
3	196	12
3	240	13
4	0	1
4	13	2
4	24	3
4	50	4
4	69	5
4	70	6
4	84	7
4	95	8
4	125	9
4	179	10
4	243	11
4	244	12
5	0	1
5	20	2
5	34	3
5	49	4
5	58	5
5	59	6
5	80	7
5	81	8
5	89	9
5	107	10
5	109	11
Table: Definers
PID	CUID	BASEFILEID	DEFINERID
25	5	5	2
Table: Includers
PID	CUID	BASEFILEID	INCLUDERID
24	2	2	1
24	4	3	1
24	4	4	1
25	2	2	1
25	2	2	1
25	4	3	1
25	4	4	1
25	5	3	1
25	5	5	1
Table: Providers
PID	CUID	PROVIDERID
24	2	2
24	4	4
25	2	2
25	2	2
25	4	4
25	5	5
Table: IncTriggers
PID	CUID	BASEFILEID	DEFINERID	FOFFSET	LEN
25	5	5	2	471	4
Table: Functions
ID	NAME	ISMACRO	DEFINED	DECLARED	FILESCOPED	FID	FOFFSET	FANIN
821	foo	0	1	1	0	5	81	0
4715	main	0	0	1	0	2	471	0
5355	_cscout_dummy1	0	1	1	1	2	535	1
Table: FunctionDefs
FUNCTIONID	FIDBEGIN	FOFFSETBEGIN	FIDEND	FOFFSETEND
821	5	88	5	108
5355	2	557	2	577
Table: FunctionMetrics
FUNCTIONID	PRECPP	NCHAR	NCCOMMENT	NSPACE	NLCOMMENT	NBCOMMENT	NLINE	MAXLINELEN	MAXSTMTLEN	MAXSTMTNEST	MAXBRACENEST	MAXBRACKNEST	BRACENEST	BRACKNEST	NULINE	NPPDIRECTIVE	NPPCOND	NPPFMACRO	NPPOMACRO	NTOKEN	NSTMT	NOP	NUOP	NNCONST	NCLIT	NSTRING	NPPCONCATOP	NPPSTRINGOP	NIF	NELSE	NSWITCH	NCASE	NDEFAULT	NBREAK	NFOR	NWHILE	NDO	NCONTINUE	NGOTO	NRETURN	NASM	NTYPEOF	NPID	NFID	NMID	NID	NUPID	NUFID	NUMID	NUID	NLABEL	NMACROEXPANDTOKEN	NGNSOC	NMPARAM	NFPARAM	NEPARAM	FANIN	FANOUT	CCYCL1	CCYCL2	CCYCL3	CSTRUC	CHAL	IFLOW
821	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	0	0	0	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	1	0	0	0	0	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	NULL	0	0	0	NULL	0	0	NULL	20	NULL	0	NULL	0	0	1	1	1	0.0	0.0	0.0
821	1	21	14	3	1	0	3	17	0	NULL	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	NULL	0	NULL	0	0	0	1	1	1	0.0	0.0	0.0
5355	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	5	0	0	1	NULL	NULL	NULL	NULL	NULL	NULL	NULL	5	1	0	0	0	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	NULL	1	0	1	NULL	1	0	NULL	12	NULL	0	NULL	1	1	1	1	1	1.0	0.0	1.0
5355	1	22	0	3	0	0	1	21	5	NULL	0	1	0	0	0	0	0	0	0	5	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	1	0	1	0	1	0	0	NULL	0	NULL	0	1	1	1	1	1	1.0	0.0	1.0
Table: FunctionId
FUNCTIONID	ORDINAL	EID
821	0	821
4715	0	421
5355	0	5355
Table: Fcalls
SOURCEID	DESTID
5355	5355
Done
